From 1604147422e64b8ba001a88b1b5d1f507aeb42dc Mon Sep 17 00:00:00 2001
From: Christian Ege <k4230r6@gmail.com>
Date: Sat, 31 Dec 2016 06:46:15 +0100
Subject: [PATCH] Added device tree overlay support

This will provide support for the long awaited device tree support

Signed-off-by: Christian Ege <k4230r6@gmail.com>
Signed-off-by: Drew Moseley <drew.moseley@northern.tech>
---
 configs/udoo_a62_defconfig       | 1 +
 configs/udoo_neo_defconfig       | 1 +
 configs/udoo_quad_dual_defconfig | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/udoo_a62_defconfig b/configs/udoo_a62_defconfig
index 1b35fc963a..b02fb02c99 100644
--- a/configs/udoo_a62_defconfig
+++ b/configs/udoo_a62_defconfig
@@ -35,3 +35,4 @@ CONFIG_OF_LIBFDT=y
 CONFIG_SPI_FLASH=y
 CONFIG_APX_WATCHDOG=y
 CONFIG_MXC_SPI_CSHOLD=y
+CONFIG_OF_LIBFDT_OVERLAY=y
diff --git a/configs/udoo_neo_defconfig b/configs/udoo_neo_defconfig
index e8df11db66..6bbf7f3db2 100644
--- a/configs/udoo_neo_defconfig
+++ b/configs/udoo_neo_defconfig
@@ -34,3 +34,4 @@ CONFIG_PHY_MICREL=y
 CONFIG_PHY_MICREL_KSZ8XXX=y
 CONFIG_MII=y
 CONFIG_OF_LIBFDT=y
+CONFIG_OF_LIBFDT_OVERLAY=y
diff --git a/configs/udoo_quad_dual_defconfig b/configs/udoo_quad_dual_defconfig
index 02337ef64e..c9d201ca20 100644
--- a/configs/udoo_quad_dual_defconfig
+++ b/configs/udoo_quad_dual_defconfig
@@ -30,3 +30,4 @@ CONFIG_CMD_FS_GENERIC=y
 CONFIG_DM=y
 CONFIG_DM_THERMAL=y
 CONFIG_OF_LIBFDT=y
+CONFIG_OF_LIBFDT_OVERLAY=y
-- 
2.22.0

